packages/eslint-config/testing/react.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  extends: ['plugin:testing-library/react'],
+  plugins: ['testing-library'],
```

Comment:
I don't think I understand how these files work -- why does this have a plugins array but none of the other ones have one?

Reply:
I think these plugins should only be here if we're using a rule from them, not if we're loading in a full config with `extends`. I'll clean these up.